src/lambda/local/debugConfiguration.ts

```diff
@@ -213,14 +168,17 @@ export function getTemplateResource(
     if (!cfnTemplate) {
         throw Error(`template not found (not registered?): ${templateInvoke.templatePath}`)
     }
-    if (!cfnTemplate?.Resources) {
+    const res = cfnTemplate.Resources
+    if (!res) {
         throw Error(`no Resources in template: ${templateInvoke.templatePath}`)
     }
-    const templateResource: CloudFormation.Resource | undefined = cfnTemplate?.Resources![templateInvoke.logicalId!]
+    const templateResource: CloudFormation.Resource | undefined =
+        config.invokeTarget.target === 'api'
+            ? res[CloudFormation.getResourceByApiPath(res, config.api?.path ?? '')]
+            : res[templateInvoke.logicalId!]
     if (!templateResource) {
         throw Error(
-            `template Resources object does not contain key '${templateInvoke.logicalId}':` +
-                ` ${JSON.stringify(cfnTemplate?.Resources)}`
+            `template Resources object does not contain key '${templateInvoke.logicalId}':` + ` ${JSON.stringify(res)}`
```

Comment:
nit: Is there a reason to concatenate two template literals?

Reply:
that was done by prettier. but now I will undo it.